### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by gohdaniel15**
This PR updates the default github README template with the Tinkerbox README template.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,67 @@
...
+2.Install gem dependencies
 
-* How to run the test suite
+  ```bash
+  cd ac_project_sample
+  bundle install
+  ```
 
-* Services (job queues, cache servers, search engines, etc.)
+4. Setup Database
```
Comment (hadi91) [inline README.md]: It should be 3, duh.

File: README.md [hunk 2]
```diff
@@ -1,24 +1,67 @@
...
+5. Start the application
+
+  ```bash
+  # Start the web server
+  rails server
+  ```
+
+## Testing
+
+We use [Rspec](https://github.com/rspec/rspec-rails) as our tetsing framework. So, during development, run these regularly.
```
Comment (hadi91) [inline README.md]: Spelling should be testing ISO tetsing.

**Review by hadi91 — CHANGES_REQUESTED** [review]
Pls change the errors stated.

File: README.md (hunk 1 above)
Comment (gohdaniel15) [reply README.md]: thanks bruh!

File: README.md (hunk 2 above)
Comment (gohdaniel15) [reply README.md]: Haha silly me 😛

**Review by hadi91 — APPROVED** [review]
Awesome. LGTM 👍

**Pull request merged**